chapters/05-version-control-solo.Rmd

```diff
@@ -12,22 +12,49 @@ Clearly identify which tasks are best accomplished by each interface.
   - adding and updating remotes
 - Setting up GitHub for R package (make use of `usethis`), pushing and pulling
 
-## Final exercise
+<!---
+Moved from syllabus:
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+And some people still worry that if they make their data and code generally available,
+someone else will use it and publish a result they have come up with themselves.
+This is almost unheard of in practice,
+but that doesn't stop people using it as a scare tactic.
+Other people are afraid of looking foolish or incompetent by sharing code that might contain bugs.
+This isn't just [impostor syndrome][impostor-syndrome]:
+members of marginalized groups are frequently judged more harshly than others,
+so being wrong in public is much riskier for them.
+-->
 
```

Comment:
I'm not sure this language fits in this chapter. I think we make a standard statement saying "we're publishing code," which can normalize it, and then reference the community involvement chapter later.